react-backend/app.js

```diff
@@ -42,6 +43,7 @@ app.use('/activities', activities);
 app.use('/checkCB', checkCB);
 app.use('/registerCB', registerCB);
 app.use('/checkCBlogin', checkCBlogin);
+app.use('/checkCBemail', checkCBemail);
```

Comment:
also needs to be updated to `CBPasswordResetInstigator` here (or whatever you decide to call it when you require it in above).